src/util.js

```diff
@@ -495,10 +495,24 @@ Candy.Util = (function(self, $){
 		 *   Emotified text
 		 */
 		emotify: function(text) {
-			var i;
-			for(i = this.emoticons.length-1; i >= 0; i--) {
+			var codeMatches = text.match(/(\`[^`]*\`)/gim) || [];
+			var codeHolders = {
```

Comment:
I don't think that add/replace are particularly more descriptive here. Perhaps a brief comment is necessary?